questionpy_sdk/webserver/app.py

```diff
@@ -51,11 +52,12 @@ async def render_options(request: web.Request) -> web.Response:
 
     async with webserver.worker_pool.get_worker(webserver.package, 0, None) as worker:
         manifest = await worker.get_manifest()
+        form_definition: OptionsFormDefinition
         form_definition, form_data = await worker.get_options_form(old_state)
```

Comment:
Hier (und unten) könnte man auch den `worker` typehinten, um JetBrains auf die Sprünge zu helfen. 

```suggestion
    async with webserver.worker_pool.get_worker(webserver.package, 0, None) as worker:
        worker: Worker
        manifest = await worker.get_manifest()
        form_definition, form_data = await worker.get_options_form(old_state)
```